examples/api/custom_projection_example.py

```diff
@@ -3,7 +3,7 @@
 Custom projection
 =================
 
-This example showcases the Hammer projection by alleviating many features of
+Hammer projection by alleviating many features of
```

Comment:
This sentence fragment no longer makes sense like this. Though to be honest, I don't think "alleviating" was the right word in the first place.